samples/CMakeLists.txt

```diff
@@ -79,6 +79,8 @@ if( EXPAT_FOUND )
             exiv2-xmp
             ${EXPAT_LIBRARIES}
     )
+    target_include_directories(geotag PRIVATE ${CMAKE_BINARY_DIR})          # exv_conf.h 
```

Comment:
Hi Robin, I think these two lines are not needed.

We are including the  `${CMAKE_BINARY_DIR}` globally for all the project targets at `CMakeLists.txt:78`.

The `${CMAKE_SOURCE_DIR}/include` directory is also propagated when we add the dependency of each sample target to the library exiv2:
- On `samples/CMakeLists.txt:128` we are adding the dependency to the **exiv2** target.
- On `src/CMakeLists.txt:150-154` we indicated the directoryes which are propagated when some target depends on **exiv2.**

Reply:
https://github.com/Exiv2/exiv2/issues/1516#issuecomment-814164414

Reply:
oh I see, I'll try to reproduce it on my side ... It should see first the headers specified in the project rather than something you have in /usr/local/inclde